### PR TITLE
Fixes #2853 Apply new heading to all Pico devices

### LINKED ISSUE #2853: [Pico] The on screen controls prompt is not displayed in the view where it's triggered
**[Affected Versions]:**
- Firefox Reality 9 RC3 (18aba8b)

**[Affected Platforms]:**
- Pico Neo 2 - Android - 8.1.0

**[Prerequisites]:**
- The headset is turned on and the app is opened.

**[Steps to reproduce]:**										
1. Open a 360 video and play it in immersive mode.
2. Move your head to the right/left or back.
3. Press the trigger button from the controller.
4. Observe the behavior. 

**[Expected results]:**
- The on screen controls are displayed in your view.

**[Actual results]:**
- The on screen controls are always displayed in the front view even if it was triggered in a different position.

**[Notes]:**
- Attached is a screen recording of the issue: [link](https://drive.google.com/open?id=1N8xd8HxlAZAQrZE5RgntMqStAVYFrU7G).

### PR DESCRIPTION
**Pull request by keianhzo**
Fixes #2853 Apply new heading to all Pico devices

### CODE REVIEW

**Comment by bluemarvin** [comment]
Looks like too many commits were included in this PR?

**Comment by keianhzo** [comment]
@bluemarvin updated, I think I mistakenly started this PR from another branch.

File: app/src/picovr/cpp/DeviceDelegatePicoVR.cpp [hunk 1]
```diff
@@ -288,9 +288,7 @@ DeviceDelegatePicoVR::GetReorientTransform() const {
 
 void
 DeviceDelegatePicoVR::SetReorientTransform(const vrb::Matrix& aMatrix) {
-  if (m.type == kTypeNeo2) {
-    m.reorientMatrix = aMatrix;
-  }
+  m.reorientMatrix = aMatrix;
```
Comment (bluemarvin) [inline app/src/picovr/cpp/DeviceDelegatePicoVR.cpp]: If you do this, re-center is broken on the G2.

**Comment by bluemarvin** [comment]
closing in favor of  #2973

**Pull request closed without merging**